src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        FileInputStream fis = null;
+        try {
+            fis = new FileInputStream(fromFileName);
```

Comment:
Remember, if you are using classes that implement an AutoCloseable interface, we should use it with try-with-resources.